diff -urN -x CVS kdevelop.orig/parts/diff/diffwidget.cpp kdevelop/parts/diff/diffwidget.cpp
--- kdevelop.orig/parts/diff/diffwidget.cpp	Sun Feb 22 05:43:52 2004
+++ kdevelop/parts/diff/diffwidget.cpp	Tue Aug 17 01:51:09 2004
@@ -155,7 +155,7 @@
   init = true;
 
   // search all parts that can handle text/x-diff
-  KTrader::OfferList offers = KTrader::self()->query("text/x-diff", "('KParts/ReadOnlyPart' in ServiceTypes) and ('text/x-diff' in ServiceTypes)");
+  KTrader::OfferList offers = KTrader::self()->query("text/x-diff", "('KParts/ReadOnlyPart' in ServiceTypes) and ('text/x-diff' in ServiceTypes) and (DesktopEntryName != 'katepart')");
   KTrader::OfferList::const_iterator it;
   for ( it = offers.begin(); it != offers.end(); ++it ) {
     KService::Ptr ptr = (*it);
