--- kdevelop-3.5.1/languages/cpp/app_templates/cppcurseshello/app-configure.in~	2005-09-10 11:22:32.000000000 +0300
+++ kdevelop-3.5.1/languages/cpp/app_templates/cppcurseshello/app-configure.in	2008-05-01 14:05:04.507204713 +0300
@@ -10,8 +10,9 @@
 AC_LANG_SAVE
 AC_LANG_CPLUSPLUS
 
-AC_CHECK_HEADER([ncurses.h], [],
-	AC_MSG_ERROR([Couldn't find ncurses headers.]))
+AC_CHECK_HEADERS([ncurses/curses.h],
+	[ CPPFLAGS="-I/usr/include/ncurses $CPPFLAGS" ],
+	[ AC_CHECK_HEADERS(curses.h,, [AC_MSG_ERROR(["curses.h not found"])],) ])
 
 AC_CHECK_LIB(ncurses, refresh, , AC_ERROR([Couldn't find ncurses library]), -lncurses, -ldl)
 
