--- kdevelop-030930/Makefile.am~	2003-09-30 01:22:35.000000000 +0200
+++ kdevelop-030930/Makefile.am	2003-10-02 11:11:15.000000000 +0200
@@ -79,10 +79,8 @@
 ## We need this until we change the application name from gideon back to kdevelop
 install-gideon-kdevelop:
 	@if test ! -d $(DESTDIR)$(kde_htmldir)/en/gideon; then \
-		pushd `pwd`;\
 		echo "*** Creating a softlink from kdevelop to gideon to make the menu item Help -> Gideon Handbook work:";\
 		cd $(DESTDIR)$(kde_htmldir)/en;\
 		echo $(LN_S) kdevelop gideon;\
 		$(LN_S) kdevelop gideon;\
-		popd;\
 	fi
